hnn_core/dipole.py

```diff
@@ -11,7 +11,7 @@
 
 
 def simulate_dipole(net, n_trials=None, record_vsoma=False,
-                    record_isoma=False, postproc=True):
```

Comment:
for some reason, in my head there was also a `None` option whereby whatever was specified in the params file was respected, is there some such thing anywhere?

Reply:
Not that I have seen.